qutebrowser/utils/utils.py

```diff
@@ -810,6 +811,11 @@ def get_clipboard(selection=False):
         mode = QClipboard.Selection if selection else QClipboard.Clipboard
         data = QApplication.clipboard().text(mode=mode)
 
+    target = "Primary selection" if selection else "Clipboard"
+    if not data.strip():
+        raise cmdexc.CommandError("{} is empty.".format(target))
```

Comment:
Sorry, somehow missed this question. Since `replace_variables` gets called inside `run` and e.g. `run_safely` calls `run` and catches `cmdexc.CommandError` it should be alright.